go/systests/team_invite_test.go

```diff
@@ -366,7 +366,7 @@ func TestImpTeamWithMultipleRooters(t *testing.T) {
 			if (arg.TeamID.Eq(team1) || arg.TeamID.Eq(team2)) && arg.Changes.MembershipChanged && !arg.Changes.KeyRotated && !arg.Changes.Renamed && arg.LatestSeqno == toSeqno {
 				t.Logf("change matched with %q", arg.TeamID)
 				winningTeam = arg.TeamID
-				return
+				break
```

Comment:
This test was busted...It doesn't pass now.

Reply:
@zapu can you take a look?

Reply:
woah, thanks for finding this. can you skip if it doesn't pass?

Reply:
Yeah, i'll skip it for now in this branch.

Reply:
i added a ticket for it